ddspipe_yaml/src/cpp/YamlReader_features.cpp

```diff
@@ -54,12 +55,27 @@ void YamlReader::fill(
     }
 }
 
+template <>
+DDSPIPE_YAML_DllAPI
+bool YamlValidator::validate<participants::XmlHandlerConfiguration>(
+        const Yaml& yml,
+        const YamlReaderVersion& /* version */)
+{
+    const std::set<TagType> tags{
```

Comment:
Instead of filling all the sets in here I would create them in ``yaml_configuration_tags.hpp``

Reply:
I'm not sure I understand. I did it here so it could be easily seen what the validation of each function does.